Child org tab information added to Accounts page (#3431)
SumoLogic/sumologic-documentation: docs/manage/manage-subscription/create-manage-orgs-service-providers.md

* **Infrequent Ingest**. Credits used for logs ingested into the Infrequent tier.
* **Infrequent Scan**. Credits used to scan data for Infrequent tier searches.
* **Tracing Ingest**. Credits used for traces ingested.
* **Metrics Ingest**. Credits used for metrics ingested.
* **Storage**. Credits used for log storage in the Continuous and Frequent tiers.
* **Cloud SIEM Ingest**. Credits used for logs ingested into Cloud SIEM.
* **Infrequent Storage**. Credits used for log storage in the Infrequent tier.
* **Promotional categories**. For more information, see [Monitoring Promotional Credit usage](cloud-flex-credits-accounts.md#monitoring-promotional-credit-usage).

<img src={useBaseUrl('img/subscriptions/account-overview-new.png')} alt="your description" style={{border:'1px solid black'}} />

### View child credit usage by category

You can view the usage of individual child orgs in the **Child Orgs** tab on the **Account Overview** page. Usage for the following categories is shown:

* The **Credit Allocation** tile indicates the total number of credits allocated.
* The **Total Credit Usage** tile shows amount of credits consumed out of the total allocated credits.

> * The **Usage Forecast** tile indicated the total consumption of the credits by the end of contract period according to the current consumption rate.

Reviewer: ```suggestion
* The **Usage Forecast** tile indicates the total consumption of the credits by the end of contract period according to the current consumption rate.
```

* The **Allocated Credits** column indicates the total number of credits allocated to the selected child org.
* The **Total Credits Used** column indicates the total amount of credits consumed for the selected child org.
* The **Usage %** column indicates the percentage of overall subscription credits that was allocated to a child org has been consumed.
* The **Forecast** column indicates the percentage of total consumption of the credits by the end of contract period according to the current consumption rate for a selected child org.
* The **Usage % WOW** column indicates the percentage of week on week subscription credits that was allocated to a child org has been consumed.

<img src={useBaseUrl('img/subscriptions/child-org.png')} alt="your description" style={{border:'1px solid black'}} />

> On clicking any of the selected Child Org, you can view the usage for the following categories:

Reviewer 2: ```suggestion
By clicking any of the selected child orgs, you can view the usage for the following categories:
```
